core/templates/dev/head/components/version_diff_visualization_directive.html

```diff
@@ -5,7 +5,7 @@
 
 <div class="row">
   <div class="col-sm-8 col-md-8 col-lg-8">
-    <div state-graph-viz class="protractor-test-history-graph diff-graph"
+    <div state-graph-visualization class="protractor-test-history-graph diff-graph"
```

Comment:
I think this directive can't be used as an attribute according to the note [here](https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/pages/exploration_editor/editor_tab/StateGraphVisualizationDirective.js#L23)

Reply:
Hm, we need to fix that at some stage. I think it will end up being part of the Angular 2 migration, so /cc @ankita240796 for info (no need to fix in this PR).